app/controllers/rooms_controller.rb

```diff
@@ -0,0 +1,141 @@
+class RoomsController < ApplicationController
+  def show
+    p current_user.show_solutions.to_s
+    @setup = Setup.take
+    @week = @setup.weeks.find_by_number(params[:week_number])
+
+    @room = Room.find_by_id(params[:room_number])
+
+    @next_room = @room.next(current_user.id,@week.id)
+    @previous_room = @room.previous(current_user.id,@week.id)
+
+    @learning_objects = @room.learning_objects.order(id: :asc).distinct
+    @results = UserToLoRelation.get_results(current_user.id,@week.id)
+
+    @user = current_user
+  end
+
+  def eval
+    @setup = Setup.take
+    @room = Room.find(params[:room_number])
+    learning_objects = @room.learning_objects.all.distinct
```

Comment:
`distinct` - moze byt v miestnosti jeden learning object viackrat? Tomu by bolo dobre zabranit... (Ako sa to generuje? Validacie?)